runners/gearpump/src/main/java/org/apache/beam/runners/gearpump/TestGearpumpRunner.java

```diff
@@ -52,7 +53,10 @@ public static TestGearpumpRunner fromOptions(PipelineOptions options) {
   @Override
   public GearpumpPipelineResult run(Pipeline pipeline) {
     GearpumpPipelineResult result = delegate.run(pipeline);
+    PipelineResult.State state = result.waitUntilFinish();
     cluster.stop();
+    assert(state == PipelineResult.State.DONE);
```

Comment:
May I add that later and skip those tests for now since Gearpump doesn't have a good way to return status to users yet ?